vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -141,11 +141,13 @@ def _get_schema(self, weight_quant: BaseModel,
         if self.quant_format == CompressionFormat.int_quantized.value:
             if self._is_static_tensor_w8a8(weight_quant, input_quant):
                 return CompressedTensorsW8A8StaticTensor(
```

Comment:
Should be updated to consider the correct format that we expect from the config file. Wont be int_quantized

Reply:
Looks like the models coming out are `int_quantized`

- https://huggingface.co/nm-testing/Meta-Llama-3-8B-FP8-compressed-tensors-test/blob/main/config.json#L40